usage/launcher/src/test/java/org/apache/brooklyn/launcher/BrooklynWebServerTest.java

```diff
@@ -146,11 +146,13 @@ public void verifyHttpsFromConfig() throws Exception {
     @Test
     public void verifyHttpsCiphers() throws Exception {
         brooklynProperties.put(BrooklynWebConfig.HTTPS_REQUIRED, true);
-        brooklynProperties.put(BrooklynWebConfig.TRANSPORT_PROTOCOLS, "XXX");
-        brooklynProperties.put(BrooklynWebConfig.TRANSPORT_CIPHERS, "XXX");
```

Comment:
These properties override the default protocols, certificates so that the handshake fails. Can't figure out why the test succeeds now - what's the reason for the exception?